### DISCUSSION TITLE
How to generate header C files?

### DISCUSSION THREAD

**Discussion by Benik3** [dvidelabs/flatcc#243] [Q&A] [answered]
Hello there.
I try for the first time FlatBuffer. I successfully built flatcc.exe on windows using Visual Studio 2019.
But I didn't find, how to generate header file for C from fbs, the flatcc.exe has only option for c++.
But here I see that C header file are somehow achievable: https://github.com/webosbrew/hyperion-webos/tree/45f2514aaa42a602b36659d627181b399ca8f365/fbs
Thank you!

**Comment by mikkelfj** [comment, marked as answer]
Hi, I think maybe you are confusing flatc and flatcc. flatc generates C++, flatcc generates C. If you follow the README in this repository, you should soon be creating header files in C. If not, please ask again. Please let my know if it works out for you.

**Reply by Benik3** [reply]
Ok, I'm just an idiot.
I had both flatc.exe and flatcc.exe in the same folder and I missed, that I run the flatc...
Sorry for bothering 🙇

**Reply by mikkelfj** [reply]
The path to success is the path of failures :-)
